packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,99 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { requireNativeComponent } from 'react-native';
+import { IViewProps } from '@fluentui-react-native/adapters';
+import * as React from 'react';
+
+const shimmerName = 'Shimmer';
+
+const NativeShimmerView = requireNativeComponent('MSFShimmerView');
+
+/**
+ * Appearance of the views that are shimmered
+ */
+export interface IShimmerViewAppearance {
+  /**
+   * Color to tint the shimmer boxes. Defaults to the Fabric default color.
+   */
+  tintColor: number;
+
+  /**
+   * Corner radius on each view.
+   */
+  cornerRadius: number;
+
+  /**
+   * Corner radius on each UILabel. Set to < 0 to disable and use default `cornerRadius`.
+   */
+  labelCornerRadius: number;
+
+  /**
+   * usesTextHeightForLabels: True to enable shimmers to auto-adjust to font height for a UILabel -- this
+   * will more accurately reflect the text in the label rect rather than using the bounding box. `labelHeight`
+   * will take precedence over this property.
+   */
+  usesTextHeightForLabels: boolean;
+
+  /**
+   * If greater than 0, a fixed height to use for all UILabels. This will take precedence
+   * over `usesTextHeightForLabels`. Set to less than 0 to disable.
+   */
+  labelHeight: number;
+}
+
+/**
+ * Appearance of the shimmer itself (the animation appearance)
+ */
+export interface IShimmerAppearance {
+  alpha: number;
+  width: number;
+
+  /**
+   * Angle of the direction of the gradient, in radian. 0 means horizontal, Pi/2 means vertical.
+   */
+  angle: number;
+
+  /**
+   * Speed of the animation, in point/seconds.
+   */
+  speed: number;
+
+  /**
+   * Delay between the end of a shimmering animation and the beginning of the next one, in seconds
+   */
+  delay: number;
+}
+
+export interface ShimmerTokens {}
```

Comment:
Should these values actually be set in tokens? Do we want each shimmer to be individually customizable or do we expect people to mostly leave them as is?

Reply:
I expect people won't be customizing shimmer past the default.

These should be tokens, in my opinion. I was having difficulty with turning them into tokens, so I wanted to start with having them as props. 

Another thing about tokens: the native iOS view sets the defaults for each of these if no appearance / shimmerAppearance is specified. We *could* export each of those from Native -> React Native and set those when creating the control with compose. That was another thing I wasn't sure about, since I didn't see a pattern to follow.

Reply:
Shimmer.customize works now. We now have a token :D